Artifacts.toml

```diff
@@ -318,6 +318,43 @@ os = "windows"
     url = "https://github.com/JuliaBinaryWrappers/CUDA_loader_jll.jl/releases/download/CUDA_loader-v0.2.0+2/CUDA_loader.v0.2.0.x86_64-w64-mingw32-cuda+11.4.tar.gz"
 
 
+# CUDA forward compatibility package
+
+[[CUDA_compat]]
+arch = "x86_64"
+cuda = "11.4"
+git-tree-sha1 = "d9b5623143a996f924e7a5eee9d06354cb650d73"
```

Comment:
How did you compute this tree sha?  In `JuliaBinaryWrappers` there is `b618eadab05abb3bef21da981a51b6f4f0183b27` https://github.com/JuliaBinaryWrappers/CUDA_compat_jll.jl/blob/bd512f429381c57f88183fbd97a7f4d10fd5df03/Artifacts.toml#L3, which is consistent with what I get in git:
```console
CUDA_compat.v11.5.50.x86_64-linux-gnu% git log --pretty=format:'%T %s'
b618eadab05abb3bef21da981a51b6f4f0183b27 first commit
```

Reply:
Pkg issue filed https://github.com/JuliaLang/Pkg.jl/issues/2929

Reply:
Probably a bad copy paste :facepalm:

Reply:
Should be fixed in https://github.com/JuliaGPU/CUDA.jl/pull/1310